web/cypress/integration/route/create-route-both-use-uri-uris.spec.js

```diff
@@ -0,0 +1,175 @@
+    cy.get(selector.uris_0).clear().type(data.uri_0);
+    cy.get(selector.remote_addrs_0).type(data.remote_addr_0);
+
+    cy.contains('Next').click();
+    cy.get(selector.nodes_0_host).type(data.host);
+    cy.get(selector.nodes_0_port).type(data.port);
+    cy.get(selector.nodes_0_weight).type(data.weight);
+    cy.contains('Next').click();
+
+    cy.contains('Next').click();
+    cy.contains('Submit').click();
+    cy.contains(data.submitSuccess);
+
+    // back to route list page
+    cy.contains('Goto List').click();
+    cy.url().should('contains', 'routes/list');
+  });
+
+  it('Raw Data should have uri/host/remote_addr', () => {
+    cy.visit('/');
+    cy.contains('Route').click();
+
+    cy.get(selector.nameSelector).type(data.name);
+    cy.contains('Search').click();
+    cy.contains(data.name).siblings().contains('More').click();
+    cy.contains('View').click();
+    cy.get(selector.drawer).should('be.visible');
+
+    cy.get(selector.monacoScroll).within(() => {
+      cy.contains('uri').should('exist');
+      cy.contains('host').should('exist');
+      cy.contains('remote_addr').should('exist');
+    });
+  });
+
+  it('should create route with uris/hosts/remote_addrs', () => {
+    cy.visit('/');
+    cy.contains('Route').click();
+
+    cy.get(selector.nameSelector).type(data.name);
+    cy.contains('Search').click();
+    cy.contains(data.name).siblings().contains('Configure').click();
+
+    cy.get('#status').should('have.class', 'ant-switch-checked');
+
+    cy.get(selector.hosts_0).should('have.value', data.host_0);
+    cy.get(selector.uris_0).should('have.value', data.uri_0);
+
+    cy.get(selector.addHost).click();
+    cy.get(selector.hosts_1).type(data.host_1);
+    cy.get(selector.addUri).click();
+    cy.get(selector.uris_1).clear().type(data.uri_1);
+    cy.get(selector.addRemoteAddr).click();
+    cy.get(selector.remote_addrs_1).type(data.remote_addr_1);
+
+    cy.contains('Next').click();
+    cy.get(selector.nodes_0_host).type(data.host);
+    cy.get(selector.nodes_0_port).type(data.port);
+    cy.get(selector.nodes_0_weight).type(data.weight);
+    cy.contains('Next').click();
+
+    cy.contains('Next').click();
+    cy.contains('Submit').click();
+    cy.contains(data.submitSuccess);
+
+    // back to route list page
+    cy.contains('Goto List').click();
+    cy.url().should('contains', 'routes/list');
+  });
+
+  it('Raw Data should have uris/hosts/remote_addrs', () => {
+    cy.visit('/');
+    cy.contains('Route').click();
+
+    cy.get(selector.nameSelector).type(data.name);
+    cy.contains('Search').click();
+    cy.contains(data.name).siblings().contains('More').click();
+    cy.contains('View').click();
+    cy.get(selector.drawer).should('be.visible');
+
+    cy.get(selector.monacoScroll).within(() => {
+      cy.contains('uris').should('exist');
+      cy.contains('hosts').should('exist');
+      cy.contains('remote_addrs').should('exist');
+    });
+  });
+
```

Comment:
please add cases to confirm the configure view render normally with `uri/host/remote_addr`
and `uris/hosts/remote_addrs`
@Baoyuantop

Reply:
resolve @liuxiran